samples/bluetooth/mesh/light_ctrl/boards/thingy53.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <init.h>
+
+#include <img_mgmt/img_mgmt.h>
+#include <os_mgmt/os_mgmt.h>
+#include <mgmt/mcumgr/smp_bt.h>
+
+#include <usb/usb_device.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(thingy53_setup);
+
+
+static int bt_smp_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+	int err = 0;
+
+	img_mgmt_register_group();
+	os_mgmt_register_group();
+
+	err = smp_bt_register();
```

Comment:
```suggestion
	int err = smp_bt_register();
```
then line 22 is unnecessary